enterprise/api_client/lms.py

```diff
@@ -434,9 +442,29 @@ def get_course_assessment_grades(self, course_id, username):
         * ``percent``: A float representing the overall grade for the course.
         * ``module_id``: The ID of the subsection.
         """
-        api_url = self.get_api_url(f"gradebook/{course_id}")
-        response = self.client.get(api_url, params={"user_contains": username})
-        response.raise_for_status()
+        attempts = 0
+        while True:
+            attempts = attempts + 1
+            api_url = self.get_api_url(f"gradebook/{course_id}")
+            try:
+                response = self.client.get(api_url, params={"username": username}, timeout=40)
```

Comment:
`username` query param is much more efficient than `user_contains` according to the aurora team.